tests/UI/campaigns/functional/BO/02_orders/01_orders/05_bulkUpdateOrdersStatus.js

```diff
@@ -41,11 +41,10 @@ Scenario:
 - Check orders new status
  */
 describe('BO - Orders : Bulk update orders status', async () => {
-  // Pre-condition: Create first order in FO
-  createOrderByCustomerTest(orderByCustomerData, baseContext);
-
-  // Pre-condition: Create second order in FO
-  createOrderByCustomerTest(orderByCustomerData, baseContext);
+  // Pre-condition: Create 2 orders in FO
+  for (let i = 1; i <= 2; i++) {
```

Comment:
Most of the time, it's better to add these keys numbers inside a named constant, it's easier in the future to find them in order to modify/adapt the test like

```
const ORDER_NUMBER = 2;
for (let i = 1; i <= ORDER_NUMBER; i++) {
```

Reply:
done thanks